pkg/deployment/resources/pod_creator.go

```diff
@@ -329,6 +329,23 @@ func createArangoSyncArgs(apiObject metav1.Object, spec api.DeploymentSpec, grou
 	return args
 }
 
+func createExporterArgs() []string {
+	options := make([]optionPair, 0, 64)
+	options = append(options,
+		optionPair{"--arangodb.jwtsecret", "$(" + constants.EnvArangodJWTSecret + ")"},
```

Comment:
Can we pass this by file? 
If not, let's extend the exporter to accept that first

Reply:
I would even prefer to not hand over the secret but a token. One could craft a specific token that allows access to statistics.

Reply:
One drawback: a special token would require an extra secret when passed by file.